src/requirements.txt

```diff
@@ -5,7 +5,7 @@ coverage==4.5.1
 flake8==3.5.0
 GeoAlchemy2==0.5.0
 geomet==0.2.0.post2
--e git+https://github.com/Amsterdam/GOB-Core.git@v0.4.3#egg=gobcore
+-e git+https://github.com/Amsterdam/GOB-Core.git@v0.4.3a#egg=gobcore
```

Comment:
This fix has been pushed into gobcore v0.4.4 so the requirement can be updated to the correct version